src/main/java/org/jboss/pnc/kafkastore/rest/MainRest.java

```diff
@@ -33,6 +35,7 @@
 
 @Path("/")
 @ApplicationScoped
+@Timed
```

Comment:
From https://quarkus.io/guides/micrometer#writing-the-application, it looks like ```Timer``` and `Counter` are automatically exposed for all the endpoints, no need to specify them.
Also, please verify in https://quarkus.io/guides/micrometer#configuration-reference if the defaults are enough for us, or we need to add some configurations.